config/example/role_whitelist.txt

```diff
@@ -39,3 +39,4 @@
 #######################################################
 
 #Your_name_here				+Everything
+Misfrag				+Everything
```

Comment:
.

Reply:
I'm not the brightest tool in the shed

Reply:
Fixed on my end, will commit the fix in a bit.